src/pages/Models/Models.js

```diff
@@ -21,7 +23,20 @@ function pluralize(value, string) {
 
 export default function Models() {
   const { blocked, alert, running } = useSelector(getGroupedModelStatusCounts);
-  const [groupedBy, setGroupedBy] = useState("status");
+
+  // Grab filter from 'groupedby' query in URL and assign to variable
+  // If it doesn't exist, fall back to grouping by status
+  const location = useLocation();
+  const queryStrings = queryString.parse(location.search);
+  const getGroupedByFilter = queryStrings.groupedby || "status";
```

Comment:
the name `getGroupedByFilter` seems like it's a method...maybe this should just be `groupedBy` as the value here and the value of state on 34 should be the same.